packages/components/dashboard/details-card/details-card.scss

```diff
@@ -0,0 +1,28 @@
+.details-card-title {
+  font-weight: var(--pf-global--FontWeight--normal);
+  font-size: var(--pf-global--FontSize--lg);
+}
```

Comment:
```suggestion
```
no need to use these classes... remove them from here as well as status card...

Reply:
I think with these it looks better - 

![image](https://user-images.githubusercontent.com/22158580/172325042-a261da94-fdb3-454d-af86-ca62365bd5ab.png)

without the heading it looks short, fat ugly

![image](https://user-images.githubusercontent.com/22158580/172325182-2433cb2d-c4ae-48ec-a0fd-157067db28d8.png)

Reply:
see the details heading

Reply:
check other dashboards... we need to keep consistency...

Reply:
okay sure then